plugins/woocommerce/src/Blocks/BlockTypes/ProductGalleryLargeImageNextPrevious.php

```diff
@@ -60,11 +60,12 @@ protected function render( $attributes, $content, $block ) {
 		}
 
 		$classes_and_styles = StyleAttributesUtils::get_classes_and_styles_by_attributes( $attributes );
+		$vertical_alignment = StyleAttributesUtils::get_align_class_and_style( $attributes );
```

Comment:
I'm not sure that it is the expected behavior, but $classes_and_styles contains the vertical alignment classes and styles too. I think that we don't want to apply them to the buttons, I think that we can do this:
```suggestion
		$classes_and_styles = StyleAttributesUtils::get_classes_and_styles_by_attributes( $attributes, array(), array("align") );
```

In this way, `get_classes_and_styles_by_attributes` will exclude the align styles.